spinalcordtoolbox/math.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+# Functions that perform mathematical operations on an image.
+
+from __future__ import absolute_import
+
+import logging
+import numpy as np
+
+from skimage.morphology import erosion, dilation, disk, ball, square, cube
```

Comment:
pyflakes doesn't understand the clever `globals()` trick:

> +./spinalcordtoolbox/math.py:10: 'skimage.morphology.erosion' imported but unused
> +./spinalcordtoolbox/math.py:10: 'skimage.morphology.dilation' imported but unused
> +./spinalcordtoolbox/math.py:10: 'skimage.morphology.disk' imported but unused
> +./spinalcordtoolbox/math.py:10: 'skimage.morphology.ball' imported but unused
> +./spinalcordtoolbox/math.py:10: 'skimage.morphology.square' imported but unused
> +./spinalcordtoolbox/math.py:10: 'skimage.morphology.cube' imported but unused

Reply:
the switch-case now fixes it